packages/rum/src/browser/requestIdleCallBack.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { requestIdleCallback } from './requestIdleCallback'
+
+describe('requestIdleCallback', () => {
+  let requestAnimationFrameSpy: jasmine.Spy
+  let cancelAnimationFrameSpy: jasmine.Spy
+  let callback: jasmine.Spy
+
+  beforeEach(() => {
+    requestAnimationFrameSpy = spyOn(window, 'requestAnimationFrame').and.callFake((cb) => {
+      cb(0)
+      return 123
+    })
+    cancelAnimationFrameSpy = spyOn(window, 'cancelAnimationFrame')
+    callback = jasmine.createSpy('callback')
+  })
```

Comment:
**💬 suggestion:** ‏restore `requestIdleCallback` and `cancelIdleCallback` in `afterEach` (like you did before)